src/Agent.Listener/Configuration.Windows/WindowsServiceControlManager.cs

```diff
@@ -35,7 +35,7 @@ public override void Initialize(IHostContext hostContext)
             _term = HostContext.GetService<ITerminal>();
         }
 
-        public void ConfigureService(AgentSettings settings, CommandSettings command)
+        public void ConfigureService(AgentSettings settings, CommandSettings command, bool preventServiceStart)
```

Comment:
Why do we need to pass preventServiceStart?

Reply:
Because of the check on line 135

Reply:
Why can't we retrieve it from command?

Reply:
We can, changed it to retrieve